<ui version="4.0" >
 <class>connect2DBDialog</class>
 <widget class="QDialog" name="connect2DBDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Connect to Database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Database</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Type:</string>
        </property>
        <property name="buddy" >
         <cstring>databaseTypeCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="databaseTypeCombo" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>&amp;Host:</string>
        </property>
        <property name="buddy" >
         <cstring>databaseHost</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="databaseHost" />
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>&amp;Port:</string>
        </property>
        <property name="buddy" >
         <cstring>databasePort</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QSpinBox" name="databasePort" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>&amp;DB Name:</string>
        </property>
        <property name="buddy" >
         <cstring>databaseName</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="databaseName" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>&amp;Username:</string>
        </property>
        <property name="buddy" >
         <cstring>databaseUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="databaseUsername" />
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>&amp;Password:</string>
        </property>
        <property name="buddy" >
         <cstring>databasePassword</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QLineEdit" name="databasePassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Table &amp;Name: </string>
        </property>
        <property name="buddy" >
         <cstring>databaseTable</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="databaseTable" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>databaseTypeCombo</tabstop>
  <tabstop>databaseHost</tabstop>
  <tabstop>databasePort</tabstop>
  <tabstop>databaseName</tabstop>
  <tabstop>databaseTable</tabstop>
  <tabstop>databaseUsername</tabstop>
  <tabstop>databasePassword</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>connect2DBDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>connect2DBDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
